services/web/server/tests/integration/02/conftest.py

```diff
@@ -12,41 +11,26 @@
 
 
 @pytest.fixture(scope="session")
-def mock_workbench_payload(tests_data_dir: Path) -> Dict:
+def fake_workbench_payload(tests_data_dir: Path) -> Dict:
     file_path = tests_data_dir / "workbench_sleeper_payload.json"
     with file_path.open() as fp:
         return json.load(fp)
 
 
 @pytest.fixture(scope="session")
-def mock_project(fake_data_dir: Path, mock_workbench_payload: Dict) -> Dict:
+def fake_project(fake_data_dir: Path, fake_workbench_payload: Dict) -> Dict:
```

Comment:
for your convention, I do not have a precise idea. I think mock data = fake data..

from [wikipedia](https://en.wikipedia.org/wiki/Mock_object), there are some funny ones:
```
In the book The Art of Unit Testing[7] mocks are described as a fake object that helps decide whether a test failed or passed by verifying whether an interaction with an object occurred. Everything else is defined as a stub. In that book, fakes are anything that is not real, which, based on their usage, can be either stubs or mocks.
```

but yeah, so let's use mock for the object (function, class, library) that we fake. then the data it uses.. actually they are sometimes fake, sometimes real data. but let's call them fake data then.

Reply:
it's about connecting the name with the libraries we use 

data for testing -> faker library , therefore name then as *fake_something*
replace return/sideffects/spy... of functions/objects for testing -> ``mocker`` plugin or ``unittest.mock`` library, therefore *mock_action*

Reply:
I don't have to like it but ok. 🙂

Reply:
@GitHK please do not hesitate to state your concerns. This is simply a convention. I explained the rationale around it. If you find some inconsistencies or have a better suggestion, just go ahead.